bin/lib/zopen-build

```diff
@@ -829,7 +829,7 @@ downloadTarBall()
   else
     freshBuild=true
     if ${verbose}; then
-      printVerbose "curl -L -o ${tarballz} ${ZOPEN_URL}"
+      printVerbose "$curlCmd -L -o ${tarballz} ${ZOPEN_URL}"
```

Comment:
Would this work for a function?

Reply:
will just print the blank string "".  Added new commit